sql/2019-07-26-19-user-affiliation-table.sql

```diff
@@ -0,0 +1,18 @@
+/**
+  * This is an SQL migration to allow storage of affiliations globally rather than per-muc-room.
+  * Data will be back-filled from chat_user_quarantine after the code is changed to read/write from this table.
+  **/
+CREATE TABLE user_affiliation (
+   id            BIGINT        NOT NULL AUTO_INCREMENT,
+   version       BIGINT        NOT NULL,
+   enabled       BIT DEFAULT 1 NOT NULL,
+   user_id       BIGINT        NOT NULL,
+   affiliation   VARCHAR(255)  NOT NULL COLLATE utf8mb4_unicode_ci,
```

Comment:
I think `text` over `varchar` is better here, due to the fact we're storing binary data and `text` is a blob